azure-pipelines.yml

```diff
@@ -171,24 +171,10 @@ stages:
           name: NetCoreInternal-Pool
           queue: buildpool.windows.10.amd64.vs2017
 
-  # Stages-based publishing entry point
-  - template: eng\common\templates\post-build\post-build.yml
-    parameters:
-      dependsOn: [PrepareForPublish]
-      # Symbol validation is not ready yet. https://github.com/dotnet/arcade/issues/2871
-      enableSymbolValidation: false
-      # Doesn't work yet, not fully configured. https://github.com/dotnet/core-setup/issues/5254
-      enableSigningValidation: false
-      # SourceLink validation doesn't work in dev builds: tries to pull from GitHub. https://github.com/dotnet/arcade/issues/3604
-      enableSourceLinkValidation: false
-      # Allow symbol publish to emit expected warnings without failing the build. Include single
-      # quotes inside the string so that it passes through to MSBuild without script interference.
-      symbolPublishingAdditionalParameters: "'-warnAsError:$false'"
-
   - template: /eng/stages/custom-publish.yml
```

Comment:
With the evolution of this, it seems like this is now 'the' publish template.  The term `custom` is adding confusion.